lms/static/js/spec/groups/views/cohorts_spec.js

```diff
@@ -359,9 +359,6 @@ define(['backbone', 'jquery', 'common/js/spec_helpers/ajax_helpers', 'common/js/
                 cohortsView.$(fileUploadForm).fileupload('add', {files: [{name: 'upload_file.txt'}]});
                 cohortsView.$('.submit-file-button').click();
 
-                // Respond to the event request
-                AjaxHelpers.respondWithNoContent(requests);
```

Comment:
Reviewers: there were several of these lines that I added in my previous PR because I was seeing an extra event. My thinking at the time was that it was an analytics event, but debugging it now I see that it was an event with a null URL. This seems to have been a bug in Sinon that has subsequently been fixed, and so this invalid extra request is no longer seen.